src/jesse_state.erl

```diff
@@ -36,9 +36,10 @@
         , set_allowed_errors/2
         , set_current_schema/2
         , set_error_list/2
-        , find_schema/2
```

Comment:
why do you remove this function ? library users may be using it, and it seems you only rename it to  `load_schema`